### PR TITLE
Removed current keymap from switch

### PR DESCRIPTION
**Pull request by NejcZdovc**
PR based on #160

### CODE REVIEW

File: src/components/popover/popover.component.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
         <layer-tab #tab *ngSwitchCase="TabName.Layer" class="popover-content" [defaultKeyAction]="defaultKeyAction"></layer-tab>
         <mouse-tab #tab *ngSwitchCase="TabName.Mouse" class="popover-content" [defaultKeyAction]="defaultKeyAction"></mouse-tab>
         <macro-tab #tab *ngSwitchCase="TabName.Macro" class="popover-content" [defaultKeyAction]="defaultKeyAction"></macro-tab>
-        <keymap-tab #tab *ngSwitchCase="TabName.Keymap" class="popover-content" [defaultKeyAction]="defaultKeyAction" [keymaps]="keymaps$ | async"></keymap-tab>
+        <keymap-tab #tab *ngSwitchCase="TabName.Keymap" class="popover-content" [defaultKeyAction]="defaultKeyAction" [keymaps]="keymaps$ | async" [currentKeymap]="currentKeymap"></keymap-tab>
```
Comment (fjozsef) [inline src/components/popover/popover.component.html]: I think the `keymaps$` should be filtered so we wouldn't need to pass the `currentKeymap` to the `keymap-tab`.

Comment (NejcZdovc) [inline src/components/popover/popover.component.html]: Done

File: src/components/popover/popover.component.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { Component, EventEmitter, Input, OnInit, Output, ViewChild } from '@angu
 
 import { Store } from '@ngrx/store';
 
+import 'rxjs/add/operator/filter';
```
Comment (fjozsef) [inline src/components/popover/popover.component.ts]: You are filtering an array, not an observable, so this is unused.

Comment (NejcZdovc) [inline src/components/popover/popover.component.ts]: It's a left over. Fixed

**Pull request merged**